content/post/2019/12/2d-platformer-demo-rework/index.md

```diff
@@ -0,0 +1,47 @@
+---
+title: "Official 2D Platformer Demo Rework"
```

Comment:
Include the product (i.e. Godot) in the title as the news posts aren't organized by app (Godot, Blender etc.) so we have to be specific.